src/admin/components/EditEventForm/style.less

```diff
@@ -74,9 +74,9 @@
   }
 
   .delete-button {
-    background-color: @red;
-    border-color: @red;
-    color: @white;
+    background-color: var(--theme-danger-1);
+    border-color: var(--theme-danger-1);
+    color: var(--theme-background);
     font-size: @general-body;
     font-weight: 500;
```

Comment:
nit: We use 500 and 700 a lot in the codebase as weights for normal and bold text. I figure we can make those variables. Maybe `--normal-weight/--bold-weight` or `--normal-font-weight/--bold-font-weight`. Up to you, though.